src/style.css

```diff
@@ -1 +1,105 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir-Book.ttf),
+    url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300, 400;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+
+  --text-placholder-color: #3D4E61;
+}
+
+input {
+  padding: 0;
+  border: transparent;
+}
+
+.page__search {
+  margin-top: 20px;
+}
+
+.search-bar {
+  position: relative;
+  box-sizing: border-box;
+}
+
+.search-bar__text-input {
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+
+  font-family: Avenir, sans-serif;
+  color: var(--text-placholder-color);
+  font-weight: 400;
```

Comment:
It's a default value for this, remove it - [DOC](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91411692/2f982670-8ee6-42a7-9480-aa7829c6c15d)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91411692/4fed5ff0-ca45-4891-ba0c-137ad079b91f)

```suggestion
```